resource_obmcs_core_subnet.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/hashicorp/terraform/helper/schema"
 
 	"github.com/oracle/terraform-provider-oci/crud"
+	"time"
```

Comment:
Should this be in the review?

Reply:
nop--I was removing the subnets "ExtraWaitPostCreateDelete" delay since that is one of the bigger test slowdowns and it munged the imports. Side note, we should investigate if that delay is still required for subnets--vcn team had a lifecycle consistency fix for that but removed it